template/src/ontology/Makefile.jinja2

```
@@ -68,6 +68,7 @@ SPARQL_VALIDATION_CHECKS =  {% for x in project.robot_report.custom_sparql_check
 SPARQL_EXPORTS =            {% for x in project.robot_report.custom_sparql_exports|default(['basic-report', 'class-count-by-prefix', 'edges', 'xrefs', 'obsoletes', 'synonyms']) %}{{ x }} {% endfor %}
 ODK_VERSION_MAKEFILE =      {% if env is defined %}{{env['ODK_VERSION'] or "Unknown" }}{% else %}"Unknown"{% endif %}
 RELAX_OPTIONS =             --include-subclass-of true
+REDUCE_OPTIONS =            --include-subproperties true
```

This will need quite a lot of debate; just FYI. Not saying anything for or against it, just putting out a warning of changing the default behaviour of reduce in ODK.

---

(I know the discussion - but this is a big change; I would suggest to keeping the current default and making it configurable to include the new behaviour)

---

You realise that _nobody_ is looking at what options are available in the ODK, do you?

Keeping the old behaviour by default and hiding the new one behind an option is a sure way to guarantee that all ontologies will keep the default behaviour (well, except the FlyBase ones), because their owners won’t even know they can change it.

That ROBOT ignores SubObjectPropertyOf axioms when reducing is a _bug_; it has been [reported](https://github.com/ontodev/robot/issues/1014) [as such](https://github.com/ontodev/robot/issues/1208). I’m fixing it.

People who liked the buggy behaviour can opt in to keep it. All they have to do is to redefine `REDUCE_OPTIONS` in their custom Makefile. This will be explained in the release notes.

I am open to the idea of making it configurable at the level of the `-odk.yaml` file rather than the Makefile, but I maintain that the default behaviour should be the not-buggy one.

---

As I said, I am not arguing against this, I am only not able to wave it through without deeply thinking through the ramifications.

If the option was at least configurable at config file level, I would probably think faster :P

---

Fine, I’ll add a config-level option. I’ll add one as well for the `RELAX_OPTION` then, since we are at it.